### PR TITLE
Add brand new logo to the project

### PR DESCRIPTION
**Pull request by caneco**
Following your request in the tweet:
http://twitter.com/icanhazstring/status/1179517916263456769

Here it is your brand new logo. Hope you like it!

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
+<p align="center">
+    <img src="https://raw.githubusercontent.com/icanhazstring/composer-unused/master/art/logo" width="300" height="360" alt="composer-unused logo">
```
Comment (icanhazstring) [inline README.md]:
```suggestion
    <img src="https://raw.githubusercontent.com/icanhazstring/composer-unused/master/art/logo.png" width="300" height="360" alt="composer-unused logo">
```

I think this should be `logo.png` right? :)

Comment (caneco) [reply README.md]: Yes… sorry about that…

`:shame: :shame: :shame: 🔔`

Comment (icanhazstring) [reply README.md]: Simply accept the suggested change then. And we are ready to go 👍

**Comment by bcremer** [comment]
@caneco Really nice and fitting logo.

Before using it I would probably ping @seldaek and/or @naderman for a quick feedback if they are OK with this logo as it's close to the original one.

**Comment by caneco** [comment]
@bcremer good point… just ping them in the tweet thread:
https://twitter.com/Caneco/status/1181529220385771521

Let's hear their feedback.

**Comment by naderman** [comment]
Thank you for getting in touch about this! While we think the logo is pretty funny and really appreciate your particular project/package, from a legal perspective we can't really allow you to use a variant of our trademark as a logo for products which are not under our direct control. Essentially we'd open it up to anyone to do the same and the logo would no longer convey that the authors of Composer have some control over the quality of the project, so anyone would eventually be allowed to modify the logo in whatever way they like and stick it on something else that we really do not approve of.

So I'm sorry to have to ask this, but we would appreciate if you didn't use this logo for your package. Maybe you can come up with your own Composer figure or something similar that is distinct enough not to suggest the package was created by us?

**Comment by icanhazstring** [comment]
Thanks for clarification @naderman. So @caneco I think we have to come up with a new plan 👌

**Comment by caneco** [comment]
@naderman sorry to hear that… but it's totally understandable. Thanks for you time!

@icanhazstring I'm going think about another solution 😄

**Comment by caneco** [comment]
Logo updated 🤞 

image

**Comment by icanhazstring** [comment]
@caneco thanks, looks nice! :)
But can you still fix the `.png` file suffix which is missing in the readme?

**Comment by caneco** [comment]
@icanhazstring done :)

**Comment by icanhazstring** [comment]
Thanks @caneco !
Merged :)

**Pull request merged**